File: output/CodeGen/codenet/C++/Java/s107204188.java
public class s107204188 {
    public int findMinDifference(int[] timePoints) {
        int minA = 1000000000;
        int num = 0;
        
        int n = timePoints.length;
        
        int t,f = 0;
        
        for(int i = 0; i < n; i++){
            t = timePoints[i];
            
            if(minA > t){
                minA = t;
            }else{
                
                if(f == t - minA)num++;
                else if(f < t - minA){
                    num = 1;
                    f = t - minA;
                }
                
            }
        }
        
        return num;
    }
}

File: output/CodeGen/codenet/C++/Python/s107204188.py
#!/usr/bin/python

import sys

def main():
    n,t,f = map(int,sys.stdin.readline().split())
    A = map(int,sys.stdin.readline().split())
    
    for i in range(n):
        if A[i] < pow(10,9) + 1:
            if f == A[i] - minA:
                num += 1
            else:
                if f < A[i] - minA:
                    num = 1
                    f = A[i] - minA
                
    print num

if __name__ == '__main__':
    main()